runtime/doc/map.txt

```diff
@@ -1629,6 +1629,7 @@ completion can be enabled:
 	-complete=help		help subjects
 	-complete=highlight	highlight groups
 	-complete=history	:history suboptions
+	-complete=keymap	keyboard mappings
```

Comment:
Ditto: a note that this is only available with `|+keymap|` or similar might be good.